pyht/utils.py

```diff
@@ -88,3 +89,12 @@ def split_text(
     if text:
         output.append(text.strip())
     return output
+
+def prepare_text(text: str | List[str], remove_ssml_tags: bool = True) -> List[str]:
```

Comment:
Why do we have some preprocessing copy-pasted here ?

Reply:
Hmm? It was previously copy-pasted but now lives only here